app/uk/gov/hmrc/customs/rosmfrontend/modules/MovementsMessagesApiProvider.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customs.rosmfrontend.modules
+
+import javax.inject.{Inject, Singleton}
+import play.api.http.HttpConfiguration
+import play.api.i18n.{DefaultMessagesApiProvider, Langs}
+import play.api.{Configuration, Environment}
+
+import scala.collection.breakOut
+
+@Singleton
+class MovementsMessagesApiProvider @Inject()(environment: Environment, config: Configuration, langs: Langs, httpConfiguration: HttpConfiguration)
```

Comment:
I would change the name of the MessagesApiProvider. Not Movements, but something related with eori-common-component e.g. `EccMessagesApiProvider`

Reply:
Oops!  Lift and shift!  Will do.